src/code42cli/cmds/departing_employee.py

```diff
@@ -12,10 +14,17 @@
 from code42cli.errors import Code42CLIError
 from code42cli.file_readers import read_csv_arg
 from code42cli.file_readers import read_flat_file_arg
+from code42cli.options import format_option
 from code42cli.options import sdk_options
 
 
 DATE_FORMAT = "%Y-%m-%d"
+filter_option = click.option(
+    "--filter",
+    help="Departing employee filter options.",
```

Comment:
Help message should specify the default option.

Reply:
`ALL` seems more relevant than 'OPEN', right now this is mirroring API's naming convention so its fine, just a thought if we could create a map of custom names against the choices.